apps/files/tests/test_utils_.py

```diff
@@ -251,8 +251,8 @@ def test_instanciate_without_data(self):
         with NamedTemporaryFile() as file_:
             file_.write(json.dumps(data))
             file_.flush()
-            pje = PackageJSONExtractor(file_.name)
-            assert pje.data == data
+            mje = ManifestJSONExtractor(file_.name)
+            assert mje.data == data
```

Comment:
The modifications in this file are totally unrelated to experiments (it's related to PR #839) that landed already, for webextensions).I'm going to remove them, and commit them on master (they're only typos).